docs/detections/rules-ui-manage.asciidoc

```diff
@@ -42,8 +42,6 @@ You can then modify the duplicated rules and, if required, delete the prebuilt o
 [[download-prebuilt-rules]]
 === Download latest prebuilt Elastic rules
 
-beta::[]
```

Comment:
FYI: This change isn't directly related to rule action export/import, but I noticed the label seemed outdated and confirmed with Garrett Spong that this feature is no longer considered beta.